ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -56,11 +62,21 @@ module.exports = Backbone.AssociatedModel.extend({
     const lat = this.get('mouseLat')
     const lon = this.get('mouseLon')
     const dms = `${mtgeo.toLat(lat)} ${mtgeo.toLon(lon)}`
+    // TODO: Move leaking defensive check knowledge to usng library (DDF-4335)
+    const mgrs =
+      lat > 84 || lat < -80
+        ? undefined
+        : converter.LLtoUSNG(lat, lon, usngPrecision)
+    const utmUps = converter.serializeUTMUPS(
```

Comment:
✏️ The serialization and conversion from lat-lon to UTM/UPS is available together in one call to `LLtoUTMUPS`.